deploy-cloudrun/src/cloudRun.ts

```diff
@@ -60,11 +60,7 @@ export class CloudRun {
 
   constructor(region: string, opts?: ClientOptions) {
     let projectId = opts?.projectId;
-    if (
-      !opts?.credentials &&
-      (!process.env.GCLOUD_PROJECT ||
-        !process.env.GOOGLE_APPLICATION_CREDENTIALS)
-    ) {
+    if (!opts?.credentials && !process.env.GOOGLE_APPLICATION_CREDENTIALS) {
```

Comment:
The Auth library is looking for GCLOUD_PROJECT (https://github.com/googleapis/google-api-nodejs-client/blob/9a3150bc18b9b23190a51a7ad18105b4b85769ec/README.md#request-level-options). We should keep this and export it like you are doing from setup-gcloud

Reply:
Makes sense, I didn't realize this was part of auth flow.

Reply:
Only for Apiary libraries which are different from the veneer libraries we typically use.